tests/e2e/test_resources.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+import pytest
+from playwright.sync_api import ConsoleMessage, Page, expect
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.slow
```

Comment:
Why is this slow?

Reply:
This and others E2E tests require the AMT server to start up and perform playwright test(s). This is slow compared to the unit tests.